internal/pkg/config/loader.go

```diff
@@ -41,12 +43,14 @@ func LoadFromFile(profile string, configuration interface{}) error {
 	}
 	contents, err := ioutil.ReadFile(fileName)
 	if err != nil {
+		LoggingClient.Error("could not load configuration file (%s): %v", fileName, err.Error())
 		return fmt.Errorf("could not load configuration file (%s): %v", fileName, err.Error())
```

Comment:
Can just build the string once and use it for logging as well as the returned error. Also err.Error() returns a string so you can use `%s` instead of `%v`

```
msg := fmt.Sprintf("could not load configuration file (%s): %s", fileName, err.Error())
LoggingClient.Error(msg)
return errors.New(msg)
```